pkg/apis/scheduling/v1alpha1/labels.go

```diff
@@ -19,3 +19,5 @@ package v1alpha1
 // GroupNameAnnotationKey is the annotation key of Pod to identify
 // which PodGroup it belongs to.
 const GroupNameAnnotationKey = "scheduling.k8s.io/group-name"
+
+const BackfillAnnotationKey = "scheduling.k8s.io/kube-batch-backfill"
```

Comment:
just `backfill-task` or something else, do not mention kube-batch here.

Reply:
fixed